src/main/java/org/jabref/gui/collab/ChangeDisplayDialog.java

```diff
@@ -50,8 +53,36 @@ public ChangeDisplayDialog(JFrame owner, final BasePanel panel,
         tree = new JTree(root);
         tree.addTreeSelectionListener(this);
 
+        //start fix issue 280
+        this.setMinimumSize(new Dimension(400, 400));
+        JPanel treePanel = new JPanel(new BorderLayout());
+        JPanel buttonAlignment = new JPanel(new FlowLayout(FlowLayout.LEADING));
+        treePanel.add(tree, BorderLayout.CENTER);
+        JButton selectAll = new JButton(Localization.lang("Select all"));
+        JButton deselectAll = new JButton(Localization.lang("Deselect all"));
+        ActionListener selectEvent = (event) -> {
```

Comment:
Maybe the event should be called "selectOrDelesectEvent" to mirror that it is called by both select and deselect button.